web/docs/getting-started.md

```diff
@@ -123,7 +123,7 @@ That's it :tada:! You have successfully created and served a new web app at <htt
 
 ## 4. What next?
 
-**Check out the 🤓 [Todo App tutorial](tutorials/todo-app.md) 🤓 , which will take you through all the core features of Wasp!**
+**Check out the 🤓 [Pick a Tutorial page](pick-a-tutorial.md) 🤓. Choose an app, that you'd like to build. And it will take you through all the core features of Wasp!**
```

Comment:
The point is to allow them to pick a tutorial that interests them. For example, I usually dislike developing to-do apps, but for someone else, it might be better to build an app they can use instead of a joke application.

Reply:
Yeah, valid point. I've updated "Pick a Tutorial" page with choices and a more detailed explanation